lib/src/Pos/Launcher/Configuration.hs

```diff
@@ -68,6 +73,40 @@ instance FromJSON Configuration where
 instance ToJSON Configuration where
     toJSON = genericToJSON defaultOptions
 
+data WalletConfiguration = WalletConfiguration
+    { ccThrottle :: !(Maybe ThrottleSettings)
+    } deriving (Show, Generic)
+
+defaultWalletConfiguration :: WalletConfiguration
+defaultWalletConfiguration = WalletConfiguration
+    { ccThrottle = Nothing
+    }
+
+instance FromJSON WalletConfiguration where
+    parseJSON = genericParseJSON defaultOptions
+
+instance ToJSON WalletConfiguration where
+    toJSON = genericToJSON defaultOptions
+
+data ThrottleSettings = ThrottleSettings
+    { tsRate   :: !Word64
+    , tsPeriod :: !Word64
+    , tsBurst  :: !Word64
+    } deriving (Show, Generic)
+
+defaultThrottleSettings :: ThrottleSettings
+defaultThrottleSettings = ThrottleSettings
+    { tsRate = 30
+    , tsPeriod = 1000000
```

Comment:
maybe set to 6sec? see https://github.com/input-output-hk/cardano-sl/blob/develop/wallet/src/Pos/Wallet/Web/Methods/Payment.hs#L80